wpilibc/athena/src/PIDController.cpp

```diff
@@ -591,12 +591,14 @@ void PIDController::InitTable(std::shared_ptr<ITable> subtable) {
  * @return Error for continuous inputs.
  */
 double PIDController::GetContinuousError(double error) const {
-  if (m_continuous &&
-      std::fabs(error) > (m_maximumInput - m_minimumInput) / 2) {
-    if (error > 0) {
-      return error - (m_maximumInput - m_minimumInput);
-    } else {
-      return error + (m_maximumInput - m_minimumInput);
+  if (m_continuous) {
+    double mod_error = std::fmod(error, m_inputRange);
```

Comment:
const qualification of member functions makes member variables immutable within that scope. Variables to which pointer or reference member variables point, local variables, and non-member variables are still mutable.